_RELEASE/Assets/assets.json

```diff
@@ -21,6 +21,7 @@
 		"go.ogg",
 		"levelUp.ogg",
 		"openHexagon.ogg",
+		"error.ogg",
```

Comment:
What's the license of this sound file? Is it free to use in a commercial project?

Reply:
This was a quick sound that I have whipped up in Audacity, where I took a source sound and edited it to make an error sound (it was just a singular ding which I modified). The source sound I got the audio from claims the audio is under Public Domain, which means even if it isn't transformative enough we cannot be sued for copyright infringement because there isn't one.

Reply:
Good enough for me. Cheers.